### PR TITLE
chore(global): Unify Github and npm namespace to `compassion-global-experience`.

### PR DESCRIPTION
**Pull request by kendrick**
@kidroca, everything seems to be fine after re-namespacing to match npm, but I'm not completely clear on the auto-publishing GitHub Action you mentioned a couple of weeks ago—could you take a quick look to validate it's not negatively impacted? Thank you!

### CODE REVIEW

**Comment by kendrick** [comment]
Looks like we might only need to update [this line in the Publish action](https://github.com/compassion-global-experience/compassion-design-system/blob/9eacc97da7537d0ad36ac45a7ae05e53dcb0db16/.github/workflows/publish.yml#LL37C11-L37C11)?

The npm org, for reference: https://www.npmjs.com/org/compassion-global-experience

File: packages/core/package.json [hunk 1]
```diff
@@ -68,5 +73,6 @@
     "storybook": "^7.0.2",
     "style-dictionary": "^3.7.2",
     "token-transformer": "0.0.32"
-  }
+  },
+  "gitHead": "b6bde2710dac0160e4df9e77ccd0ca0ca67d18e9"
```
Comment (kidroca) [inline packages/core/package.json:77]: This is line is a temp publish artifact and shouldn't be committed 
I don't know why you have it - it should be automatically removed after publishing

File: packages/react/package.json [hunk 2]
```diff
@@ -95,5 +101,6 @@
   "peerDependencies": {
     "react": "16.8.x || 17.x || 18.x",
     "react-dom": "16.8.x || 17.x || 18.x"
-  }
+  },
+  "gitHead": "b6bde2710dac0160e4df9e77ccd0ca0ca67d18e9"
```
Comment (kidroca) [inline packages/react/package.json:105]: Same as the comment on the other package

**Review by kidroca — COMMENTED** [review]
It would be better if this PR is targeted after the [221-targeted-response](https://github.com/compassion-global-experience/compassion-design-system/tree/221-targeted-response) branch, because we have some more recent changes there 

Merging this to `221` would save us from merge conflicts or re-applying changes to components only available in the latest branch

It seems like this is mostly a mass rename, so could you apply on the suggested branch?
_(I would then merge it to development as well)_

**Pull request closed without merging**